packages/sources/canton-functions/src/index.ts

```diff
@@ -0,0 +1,13 @@
+import { expose, ServerInstance } from '@chainlink/external-adapter-framework'
+import { Adapter } from '@chainlink/external-adapter-framework/adapter'
+import { config } from './config'
+import { cantonData } from './endpoint'
+
+export const adapter = new Adapter({
```

Comment:
Base contract should not have an instance, instance should be configured within implementing contracts as discussed